mesos_slave/README.md

```diff
@@ -1,33 +1,57 @@
 # Mesos_slave Integration
 
-## Overview
+# Overview
 
-Get metrics from the Mesos slave API in real time to:
+This Agent check collects metrics from Mesos slaves for:
 
-* Visualize your Mesos cluster performance
-* Correlate the performance of Mesos with the rest of your applications
+* System load
+* Number of tasks failed, finished, staged, running, etc
+* Number of executors running, terminated, etc
 
-## Installation
+And many more.
 
-Install the `dd-check-mesos_slave` package manually or with your favorite configuration manager
+This check also creates a service check for every executor task.
 
-## Configuration
+# Installation
 
-On slave nodes, configure the Agent to connect to Mesos slave's API endpoint
-Edit the `mesos_slave.yaml` file to point to your server and port
+Follow the instructions in our [blog post] to install the Datadog Agent on each Mesos agent node via the DC/OS web UI.
```

Comment:
Whoops, I forgot the link (https://www.datadoghq.com/blog/deploy-datadog-dcos/).